manifests/charts/dash/templates/deployment.yaml

```diff
@@ -26,6 +26,25 @@ spec:
         {{- include "dash.selectorLabels" . | nindent 8 }}
     spec:
       serviceAccount: dash-sa
+      initContainers:
+        - name: wait-for-postgresql
+          image: postgres:11.10
```

Comment:
Need to make this image name dynamic like other images because if they are deploying on-premise or in a firewalled environment they won't be able to pull from dockerhub (perhaps use same image as postgres image variable)

Reply:
We would need to introduce a new variable such as postgresInitImage.repository and postgresInitImage.tag in the dash chart itself if we want to do this. With the postgres being in a sibling level dependency chart under the m9sweeper install we cannot get the value from it directly as it is not in the scope for when it is parsing the values for the dash portion. We also would not want to do that because if they disable the bundled postgres installation it again would not be able to find the image.

I can add a section to the dash chart though for the postgres init container.

Reply:
Yeah, just need some way for them to override it. Maybe its a separate variable, then. 

The other option would be to use the dash image and just check postgresql is online using nodejs instead of introducing a new iamge.

Reply:
One thing we've been doing is trying to reduce image count. Makes deploying it on-premise way easier when we don't have more images than we need.

Reply:
It now includes a new CLI command to check the database status and uses the dash image already defined.